### ISSUE TITLE
Placeholder Internet Explorer

### ISSUE THREAD

**Issue by dulcetti** [postcss/autoprefixer#1003]
Hi guys.

I putted the autoprefixer with postcss in my componente. In development files work without problems. But when I build the component to the production version, with minified files, Internet Explorer execute with bugs on placeholder.

The dev version is:
```css
.field-container > .field-group > .field:-ms-input-placeholder {
  color: rgba(255, 255, 255, 0);
  transition: color .1s; }
.field-container > .field-group > .field::-ms-input-placeholder {
  color: rgba(255, 255, 255, 0);
  transition: color .1s; }
```
Working fine, but when I put both selectors with comma, Internet Explorer don't work:
```css
.field-container > .field-group > .field:-ms-input-placeholder,
.field-container > .field-group > .field::-ms-input-placeholder {
  color: rgba(255, 255, 255, 0);
  transition: color .1s; }
```
If I put only first pseudo-element with one `:`, works fine:
```css
.field-container > .field-group > .field:-ms-input-placeholder {
  color: rgba(255, 255, 255, 0);
  transition: color .1s; }
```
I can disabled the version `::-ms-input-placeholder` inside autoprefixer?

The bug runs on Internet Explore 10 and 11. Edge works fine.

**Comment by ai** [comment]
What is your input CSS?

You should write:

```pcss
.field-container > .field-group > .field::placeholder {
  color: rgba(255, 255, 255, 0);
  transition: color .1s; }
```

**Comment by dulcetti** [comment]
Sorry, I work with `SASS` and the is:
```css
.field-container
    > .field-group
        > .field
            &::placeholder
                color: rgba(#fff, 0)
                transition: color .1s
```
Yes, the generated code is:
```css
.field-container > .field-group > .field::-webkit-input-placeholder {
        color: rgba(255, 255, 255, 0);
        transition: color .1s; }
      .field-container > .field-group > .field:-ms-input-placeholder {
        color: rgba(255, 255, 255, 0);
        transition: color .1s; }
      .field-container > .field-group > .field::-ms-input-placeholder {
        color: rgba(255, 255, 255, 0);
        transition: color .1s; }
      .field-container > .field-group > .field::placeholder {
        color: rgba(255, 255, 255, 0);
        transition: color .1s; }
```
This version works fine, but, the production version is minified, and don't work:
```css
.field-container>.field-group>.field::-webkit-input-placeholder{color:hsla(0,0%,100%,0);transition:color .1s}.field-container>.field-group>.field:-ms-input-placeholder,.field-container>.field-group>.field::-ms-input-placeholder{color:hsla(0,0%,100%,0);transition:color .1s}.field-container>.field-group>.field::placeholder{color:hsla(0,0%,100%,0);transition:color .1s}
```

**Comment by ai** [comment]
You have a problem with CSS minifier, not with Autoprefixer ;). Check what minifier do you use (webpack by default uses `cssnano`) and ask them to add a special case to hot join `placeholder` prefixes.

For temporary solution (while they will fix the bug) change minifier.

**Comment by dulcetti** [comment]
@ai  Hm, ok. Thanks ;)

**Comment by tkrotoff** [comment]
For people passing by, cssnano related issues: https://github.com/cssnano/cssnano/issues/419, https://github.com/cssnano/cssnano/pull/480. It has been fixed in cssnano >= v4.0.0: https://github.com/cssnano/cssnano/releases/tag/4.0.0
